tests/sandbox/headers/test_cors.py

```diff
@@ -11,7 +11,26 @@
 @pytest.mark.parametrize("method", METHODS)
 def test_cors_options(nhsd_apim_proxy_url, method):
     """
-    .. py:function:: Test OPTIONS CORS response
+    .. py:function:: Scenario: A user submitting a request with cors \
+        headers receives a response reflecting the cors headers values
+
+        CORS allows the API to be used whilst within a web browser, from websites \
```

Comment:
Can we duplicate this in the test below? Maybe want to generate a partial so we can include it in the other CORS tests on different environments aswell?